vivo-project/Vitro
# [VIVO-1930] - Update default.runtime.properties and default.applicationSetup.n3

**[JIRA Issue](https://jira.lyrasis.org/browse/VIVO-1930)**: VIVO-1930
Companion pull request on the VIVO side is here: https://github.com/vivo-project/VIVO/pull/194

# What does this pull request do?
Improves example.runtime.properties so installation is easier for new users
Updates default database to TDB to match current best practices used for VIVO

# What's new?
1. Changes a couple default values in example.runtime.properties so first startup is smoother. 
&nbsp;&nbsp;&nbsp;&nbsp;- Change default Solr URL to match most common installation URL
&nbsp;&nbsp;&nbsp;&nbsp;- Remove email server defaults so VIVO doesn't throw warning on first startup.
&nbsp;&nbsp;&nbsp;&nbsp;~~- Change default root user email address to match VIVO's (and documentation).~~
2. Changes default database to TDB (from SDB) as we have done for VIVO

# How should this be tested?
On a new Vitro install, you should be able to ~~copy example.runtime.properties into a runtime.properties file and VIVO should~~ start up without errors or warnings (assuming you are using TDB and Solr exists at the typical location).

# Interested parties
@VIVO-project/vivo-committers
